setup-environment-internal

```diff
@@ -293,6 +309,9 @@ EOF
 if [ ! -v LMP_VERSION ]; then
     # we don't need this anymore
     unset LMP_VERSION_CACHE
+    unset LMP_PARTNER_NAME
```

Comment:
The reason for the LMP_VERSION_CACHE there is because it affects the ci build as they also used there. For this new variable we don't need to unset as they are not used on CI

Reply:
fixed